app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,6 @@
   <head>
     <title>Quibble</title>
     <%= stylesheet_link_tag    "application", media: "all" %>
-    <%= stylesheet_link_tag    "materialize", media: "screen, projection" %>
```

Comment:
This file change is not the source of the issue. It's completely fine. The problem is the changes made to `app/assets/stylesheets/materialize.css`. See comments in that file.